test/e2e/cloud/gcp/cluster_upgrade.go

```diff
@@ -595,9 +595,9 @@ func traceRouteToMaster() {
 	}
 }
 
-// checkMasterVersion validates the master version
-func checkMasterVersion(c clientset.Interface, want string) error {
-	framework.Logf("Checking master version")
+// checkInstanceVersion validates the instance version
+func checkInstanceVersion(c clientset.Interface, want string) error {
```

Comment:
Are we talking to a specific instance in the control plane, or are we talking to the control plane?